libcentrifugo/metrics.go

```diff
@@ -109,6 +112,24 @@ type metricsRegistry struct {
 	mu sync.Mutex
 }
 
+func newMetricsHistogramRegistry() *hdrhistogram.HDRHistogramRegistry {
+	quantiles := []float64{50, 90, 99, 99.99}
+	var minValue int64 = 1        // as we record latencies in microseconds, min resolution 1mks
+	var maxValue int64 = 10000000 // as we record latencies in microseconds, max resolution 10s
```

Comment:
Just measured, 15 histograms:
10sec -> 1.85mb
60sec -> 2.08mb
10min -> 2.58mb

So if you agree on 60sec I'd prefer to make 60sec